scripts/semconv/templates/semantic_attributes.j2

```diff
@@ -37,6 +47,297 @@ class {{class}}:
 {# Extra line #}
     {%- endfor %}
 
+    {%- if class == "SpanAttributes" %}
```

Comment:
Were these deprecated attributes manually added or generated somehow? How come I see ones that exist here but don't exist [here](https://github.com/open-telemetry/semantic-conventions/tree/main/model/deprecated) -> like `MESSAGING_DESTINATION`?

Reply:
some of the attributes are still defined in the spec as deprecated and others were removed for good ( probably by mistake 🤷‍♀️). E.g. `net.sock.peer.name` is still [there](https://github.com/open-telemetry/semantic-conventions/blob/9790d8f54533840dd187d3ffa5c2546d9420d357/model/deprecated/network.yaml#L10C18-L10C18) among many other deprecated attributes

Reply:
Could you add a comment (maybe jinja style `{# #}` so it doesn't end up in generated files) to that effect?